Dockerfile

```diff
@@ -5,5 +5,8 @@ COPY ./requirements.txt /tmp/requirements.txt
 RUN pip install --no-cache-dir --disable-pip-version-check --requirement /tmp/requirements.txt
 COPY ./requirements.yml /tmp/requirements.yml
```

Comment:
I could remove this COPY and RUN of `requirements.yml` but kept it here for docker cache re-use